.github/workflows/tag.yml

```
@@ -0,0 +1,26 @@
+name: Release
+# This workflow helps with creating releases.
+# This job will only be triggered when a tag (vX.X.x) is pushed
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9].[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Good progress :) though it won't support `v20.15.10`. IMHO this should be as follows:

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```